tests/acceptance/stepDefinitions/publicLinkContext.js

```diff
@@ -28,14 +28,31 @@ When(
 )
 
 When('the public uses the webUI to access the last public link created by user {string}', async function (linkCreator) {
-  const lastShareToken = await sharingHelper.fetchLastPublicLinkShare(linkCreator)
-  return client.page.publicLinkFilesPage().navigateAndWaitTillLoaded(lastShareToken)
+  const lastShare = await sharingHelper.fetchLastPublicLinkShare(linkCreator)
+  if (lastShare.share_type === sharingHelper.SHARE_TYPES.public_link) {
```

Comment:
Check for permissions rather than share_type. You should not go to the file drop page if the share permissions is for edit, view or create

Reply:
Resolved by 2e5ba545